### ISSUE TITLE
No data if response has too many columns

### ISSUE THREAD

**Issue by zvonsimextetk** [MonetDB/monetdb-nodejs#36]
I have an issue where when I select on all columns I get result like this:
![image](https://github.com/MonetDB/monetdb-nodejs/assets/174332133/ccebcfda-58a5-4fb1-bd66-70b373ed8db4)

Just regular query like "Select * from table where age=23;"

But when I select on 2 or 3 columns I get the data:
![image](https://github.com/MonetDB/monetdb-nodejs/assets/174332133/8c9a5943-092f-442d-a2bf-70ca425a9109)

Is there a limit to column number when I query? I didn't find info on this from docs.

Is there any way to remove this limit?

Thanks for help in advance

**Comment by sstalin** [comment]
Hi, 
There so no column limit. So,  doing `select * ` on 300 column wide table for example should work fine. Can you please share what version of MonetDB you are using? In the mean time I will try to reproduce the problem /add test.

**Comment by sstalin** [comment]
Can you also possibly share your table schema , or column types?

**Comment by zvonsimextetk** [comment]
Version is 11.43.21
When i use DBeaver the table is retreived and fully it seems.

I tried on another table select all and even tried selecting where i named every column and got this:
![image](https://github.com/MonetDB/monetdb-nodejs/assets/174332133/148d2ccc-4f8c-49c1-81dd-9250fd7c5366)

I tried code like this:
```
if(monetdbresult.data) {
            works.push(monetdbresult.columnCnt);
        }
```

Where monedbresult is querying different number of columns each iteration. So when it has data it registers that.


And works array looks like this are like this:
![image](https://github.com/MonetDB/monetdb-nodejs/assets/174332133/5af22f36-dc51-4bdd-b8bb-c11b3dd9e854)

Next time i tried i got different result:
![image](https://github.com/MonetDB/monetdb-nodejs/assets/174332133/89563145-4505-4a44-90b0-df1ee97c0184)


This is for table that has 56 columns. These are starting ones:
![image](https://github.com/MonetDB/monetdb-nodejs/assets/174332133/dacdcadf-4e88-4b69-be1c-3e3aecf51f19)


Rest are all integers.

Maybe it's not neccessarily a column count issue. Seems like it works consistently untill 20 columns but maybe that's coincidence?
